CHANGES.rst

```diff
@@ -46,7 +46,10 @@ ramp_fitting
 
 - Updated step docs to clarify exactly what calculations are used for
   the various flavors of variance and ERR stored in the output
-  products [#6715]
+  products [#6737]
```

Comment:
The 6737 reference needs to go down into line 52 (the entry for this PR). The reference to 6715 needs to stay as is (that was my PR for doc updates).